form.js

```diff
@@ -0,0 +1,25 @@
+function validation() {
+  const form = document.getElementById('form');
```

Comment:
Users should be able to be allowed to send messages and forms should be submitted when the email field is in lower case. Kindly see the screenshot below and fix the issue.
![lower](https://user-images.githubusercontent.com/35504149/132946749-62531dc4-3ce7-4dc8-ad3f-52ba95f7c240.png)